frontend/components/utils/FormatMarketValue.vue

```diff
@@ -19,6 +21,10 @@ export default Vue.extend({
             type: [Number, Object] as Vue.PropType<Number | BigNumber>,
             default: undefined,
         },
+        inverted: {
```

Comment:
i don't get the purpose of this variable. could you perhaps come up with a clearer name? (nit)

Reply:
This used to be how SurplusAuctions should work. Lukas had originally asked me to invert the colors (red and green) when displaying market difference, however this request we scrapped. I will remove this prop! 

Good catch!